src/pivot-table/components/cells/DataCell.tsx

```diff
@@ -57,8 +57,8 @@ const MeasureCell = ({ columnIndex, rowIndex, style, data }: MeasureCellProps):
 
   return (
     <div title={text} style={{ ...style, ...containerStyle }} data-testid={testId}>
-      <div style={cellStyle}>
-        <span style={{ ...textStyle, ...styleService.content }}>{text}</span>
+      <div style={{ ...cellStyle, display: "flex" }}>
```

Comment:
FYI I do think we need to look at styling at some point but for now this is good enough